packages/backend/src/clients/Aws/s3.ts

```diff
@@ -105,6 +109,114 @@ export class S3Service {
         return this.uploadFile(csvName, csv, 'text/csv');
     }
 
+    async uploadResults(
```

Comment:
Is this function actually results specific? It just seems like a generic upload

Reply:
I would say yes because of the configuration used in those methods (cache enabled + cache bucket) 🤔 
If we make it more abstract and pass the bucket configuration via argument we would have to consider doing the same for the other methods in this class.